### ISSUE TITLE
How i can set progress colors dynamically?

### ISSUE THREAD

**Issue by harcoPro** [nisrulz/stackedhorizontalprogressbar#4]
How i can set progress colors dynamically?

**Comment by nisrulz** [comment]
I donot think you can change them dynamically as of now. You can however set them by basically overriding the ones set in the lib.

Here is a detailed [answer on stackoverflow](https://stackoverflow.com/a/33992017/2745762)

There are ways to hack it, but I wouldn't recommend that. More so the lib is not designed to handle this situation, so you would be writing a lot of the code yourself to make it work, which defeats the purpose of using this lib. 

If you think it is required very much I can look into it sometime later, meanwhile you can look at some other lib that actually has the workaround inbuilt and functional.

If you happen to get it to work (before I work on it), please consider sending a PR to update this lib.

Thank you
